github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -203,7 +203,7 @@ presubmits:
         resources:
           requests:
             memory: "24Gi"
-  - name: pull-kubevirt-e2e-okd-4.1.0-rc.0
+  - name: pull-kubevirt-e2e-okd-4.1.0
     always_run: false
```

Comment:
Do we already want to enable it by default? If yes, `always_run` should be set to `true`.

Reply:
I think first we need to run all tests locally to see how many of the succeeds, otherwise the lane will always fail on the timeout(too many failures)
I started to run it locally but I think it will take me some time, so I do not want to delay this PR.

Reply:
I did run it locally (well, on another remote server) and it it failed with the masquerade / iptables problem I posted in the chat. I created this PR with focused tests in order to check if that is a problem on my server: https://github.com/kubevirt/kubevirt/pull/2379